src/plugins/auto/multi_schedule.cpp

```diff
@@ -251,23 +251,26 @@ MultiSchedule::~MultiSchedule() {
                     LOG_INFO_TAG("CPU_HELP:fps:%lf", count * 1000 / durtation.count());
                 }
             } else {
-                LOG_INFO_TAG("%s:infer:%ld", _workerRequest.first.c_str(), count);
-                auto n = reqAllStartTimes.size();
                 Time time;
+                // skip the first infer, Calculate fps using the time between the last infer and the second infer
+                if (!reqAllStartTimes.empty()) {
+                    reqAllStartTimes.pop_front();
+                    count--;
```

Comment:
keep count as real count. Use (count - 1) to calculate fps

Reply:
Updated.